docs/api/web-contents.md

```diff
@@ -998,6 +998,34 @@ Navigates to the specified offset from the "current entry".
 
 Returns `Boolean` - Whether the renderer process has crashed.
 
+#### `contents.crashProcess()`
```

Comment:
_(nit)_ If we made this method more descriptive about _which_ process is being crashed, the new n ame would be closer to its Chromium counterpart, e.g. `contents.forceCrashRenderer()`